pages/packingDetailed.js

```diff
@@ -289,13 +313,46 @@ class PackingOrder extends React.Component {
                   </button>
                 </React.Fragment>
               </div>
+              <h1 className="text-xl mt-4">{this.state.isPickup ? "Pickup Date" : "Delivery Date"}</h1>
+              <textarea
+                className="form-control w-full text-base font-normal text-gray-600 bg-white bg-clip-padding border border-solid border-gray-200 rounded transition ease-in-out m-0 focus:text-gray-600 focus:bg-white focus:border-blue-600 focus:outline-none"
+                id="date"
+                rows="4"
+                placeholder="Leave a note here for other pantry workers!"
```

Comment:
didn't notice this before, but let's change the placeholder to something about delivery/pickup date